authorizer/app.js

```diff
@@ -39,14 +39,17 @@ export async function handler(event) {
       SecretId: process.env.SECRET_ID,
     });
     const output = await secretsManagerClient.send(command);
-    logger.debug(output);
 
     if (output.SecretString === event.authorizationToken) {
       logger.info('event.authorizationToken match secret token');
+      output.SecretString = '<hidden>';
```

Comment:
Whilst this works, the more 'correct' way is to build a custom formatter: https://github.com/winstonjs/winston#creating-custom-formats

Also see the logger format code in our own Slack integration.

Reply:
yeah, I don't think it's necessary right now, since we only have one attribute to hide. Maybe later if we add something else.